presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetHiveRecordCursor.java

```diff
@@ -225,23 +224,7 @@ public long getTotalBytes()
     @Override
     public long getCompletedBytes()
```

Comment:
I think both are compressed size. Let me double check. If completedBytes is uncompressed size, we need to fix both Parquet and ORC @dain